chag

```diff
@@ -173,7 +214,7 @@ latest() {
 
 # Get the line number of a specific changelog entry.
 find_entry() {
-  result=$(entries_and_lines | grep -E ":${1//./\\.}$" | cut -d ':' -f 1)
+  result=$(entries_and_lines | grep "${1//./\\.}" | cut -d ':' -f 1)
```

Comment:
I haven't got a clue but it works nevertheless. To me it's working fine. Do you have any usecases that either work or break with one or the other?